dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -245,80 +170,46 @@ export const LiveBlock = ({
 		block.blockLastUpdated > block.blockFirstPublished;
 
 	return (
-		<Container id={block.id} palette={palette}>
+		<LiveBlockContainer
+			id={block.id}
+			borderColour={palette.border.liveBlock}
+		>
 			<Header>
-				<aside
-					css={css`
-						${from.phablet} {
-							/* Yes, we do need both */
-							min-width: ${ASIDE_WIDTH + GUTTER}px;
-							width: ${ASIDE_WIDTH + GUTTER}px;
-						}
-						padding-left: ${GUTTER}px;
-						padding-right: ${GUTTER}px;
-						padding-bottom: ${space[2]}px;
-					`}
-				>
-					{block.blockFirstPublished && (
-						<FirstPublished
-							firstPublished={block.blockFirstPublished}
-							blockLink={blockLink}
-						/>
-					)}
-				</aside>
-				<span>
-					{block.title && <BlockTitle title={block.title} />}
-					{headerElement &&
-						renderArticleElement({
-							format,
-							palette,
-							element: headerElement,
-							isMainMedia: false,
-							host,
-							adTargeting,
-							index: 0,
-							pageId,
-							webTitle,
-						})}
-				</span>
+				{block.blockFirstPublished && (
+					<FirstPublished
+						firstPublished={block.blockFirstPublished}
+						blockLink={blockLink}
+					/>
+				)}
+				{block.title && <BlockTitle title={block.title} />}
+				{headerElement &&
+					renderArticleElement({
+						format,
+						palette,
+						element: headerElement,
+						isMainMedia: false,
+						host,
+						adTargeting,
+						index: 0,
+						pageId,
+						webTitle,
+					})}
 			</Header>
-			<main>
-				{/* For each element, we decide what margins to set depending on the type */}
-				{mainElements.map((element, index) => {
-					if (typesWeStretch.includes(element._type)) {
-						return (
-							<BlockMedia key={`${element._type}-${index}`}>
-								{renderArticleElement({
-									format,
-									palette,
-									element,
-									adTargeting,
-									host,
-									index,
-									isMainMedia: false,
-									pageId,
-									webTitle,
-								})}
-							</BlockMedia>
-						);
-					}
-
-					return (
-						<BlockText key={`${element._type}-${index}`}>
-							{renderArticleElement({
-								format,
-								palette,
-								element,
-								isMainMedia: false,
-								index,
-								pageId,
-								webTitle,
-							})}
-						</BlockText>
-					);
-				})}
-			</main>
-			<Footer>
+			{/* For each element, we decide what margins to set depending on the type */}
+			{mainElements.map((element, index) => (
+				<p key={`${element._type}-${index}`}>
```

Comment:
Using a `p` tag here feels wrong.  In fact, having a wrapper element at all feels wrong. Looking at frontend, I can see block elements are not wrapped at all - the same as with articles.

Reply:
Hmm. How would you propose to handle the key?

Reply:
We need to remove the existence of the tag completely. Like this:

```js
			{mainElements.map((element, index) => (
				renderArticleElement({
					format,
					palette,
					element,
					isMainMedia: false,
					index,
					pageId,
					webTitle,
				})
			))}
```

Reply:
This is [how we render article elements](https://github.com/guardian/dotcom-rendering/blob/7cc306b44ad6944b2d6c03791b3d31ebeafe7504/dotcom-rendering/src/web/lib/ArticleRenderer.tsx#L58) 🤷

Reply:
For articles, we actually [add considerable complexity](https://github.com/guardian/dotcom-rendering/blob/a42df26f3e33fd7340bdea059a3f67bf91b678db/dotcom-rendering/src/web/components/elements/RewrappedComponent.tsx#L7) for the purpose of avoiding wrapping elements

That use case ^^ is not as relevant here as spacefinder will work differently but I think the principle of maintaining semantic html is still a strong motivator.

Reply:
the `index` prop will turn into a key, most of the time, when passed to `renderArticleElement`: [`renderElement.tsx` L117](https://github.com/guardian/dotcom-rendering/blob/7cc306b44ad6944b2d6c03791b3d31ebeafe7504/dotcom-rendering/src/web/lib/renderElement.tsx#L117)

Reply:
A couple of things about keys that may or may not be relevant here, based on the docs: https://reactjs.org/docs/reconciliation.html#recursing-on-children

- They recommend _not_ using array index as the key, because it can throw up issues when re-ordering elements.
- I don't think keys are that important for server-side rendering. My understanding is that they exist mainly to improve the performance of client-side re-renders. This may become more relevant if we decide to use React to handle client-side updates however.